README.md

```diff
@@ -51,7 +51,7 @@ Here is a schema illustrate the process:
 
 ## Data type representation
 
-ImJoy RPC is built on top of two-way transport layer(currently only support [Jupyter comm messages](https://jupyter-notebook.readthedocs.io/en/stable/comms.html)). Data with different types are encoded into a unified represention and send over the transport layer. It will then be decoded into the same or corresponding data type on the otherside.
+ImJoy RPC is built on top of two-way transport layer. Currently, we support 3 types of transport layer: [Jupyter comm messages](https://jupyter-notebook.readthedocs.io/en/stable/comms.html), Google Colab, and SocketIO. Data with different types are encoded into a unified represention and send over the transport layer. It will then be decoded into the same or corresponding data type on the otherside.
```

Comment:
```suggestion
ImJoy RPC is built on top of two-way transport layer. Currently, we support 3 types of transport layer: [Jupyter comm messages](https://jupyter-notebook.readthedocs.io/en/stable/comms.html), Google Colab, and SocketIO. Data with different types are encoded into a unified representation and send over the transport layer. It will then be decoded into the same or corresponding data type on the other side.
```